cupboard_app/test_queries.py

```diff
@@ -503,13 +582,23 @@ def test_create_list_ingredient(self):
         """
         Testing create_list_ingredient creates an ingredient dictionary
         """
+
+        create_custom_ingredient(self.user1.username, self.ing2.name, self.ing2.type)
+
         list_ing = create_list_ingredient(
             ingredient=self.ing1.name,
             amount=500,
             unit=self.unit1.unit
         )
         self.assertEqual(list_ing, self.list_ing1)
 
+        list_ing2 = create_list_ingredient(
+            ingredient=self.ing2.name,
+            amount=400,
+            unit=self.unit2.unit
+        )
```

Comment:
This is not testing the `create_list_ingredient` with the custom ingredient because the user id is not passed so it defaults to using the common ingredient instead of the custom ingredient.
Add the `user_id` field to ensure a list ingredient was created using the custom ingredient.